### DISCUSSION TITLE
Embed a link to another platform for two Items - voice recorded (via an API)

### DISCUSSION THREAD

**Discussion by valericac** [rubenarslan/formr.org#539] [Q&A]
Dear formr community. I hope to set up a survey that runs except for two items in formr. The two items should be voice recorded via another platform(a project running at my home institution at the University of Zürich (A speech-to-Text tool, focused on data security). Can I access and embed via an API two items in a survey and then return to the survey?

**Comment by rubenarslan** [comment]
You can use the "External" module in the run to redirect users off the formr platform. Ideally, you'll save their session code and send them back with the session code as a URL param, but it's also saved as a cookie. See "integrating" [here](https://github.com/rubenarslan/formr.org/wiki)
